### PR TITLE
inline script that works

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/views/feedback.jade [hunk 1]
```diff
@@ -17,12 +39,13 @@ html
               h2 Thanks for the feedback!
       .row
         .container
-          form#satisfaction(action="action=https://script.google.com/macros/s/AKfycbxtl0qaUunta4r6QaqSNrQookBQ6SuGN2h8VzPvn8MzK_MDDJkw/exec" method="get")
```
Comment (oatmeaI) [inline app/views/feedback.jade:20]: :joy:

**Comment by oatmeaI** [comment]
if it _has_ to be inline, :jack_o_lantern:, but it'd be nice if it wasn't

File: app/views/feedback.jade [hunk 2]
```diff
@@ -7,7 +7,29 @@ html
     meta(name='viewport' content='width=device-width, initial-scale=1')
 
     link(rel='stylesheet' href='styles/main.css')
-    link(rel='script' href='scripts/satisfaction.js')
+    script.
+      window.onload = (function() {
```
Comment (meddy) [inline app/views/feedback.jade]: `window.onload = setInputValues`
more concise
or just do this

```
window.onload = function () {
  var satisfaction = getQueryVariable('satisfaction');
  var origin = getQueryVariable('origin');
  document.getElementById('result').value = satisfaction;
  document.getElementById('origin').value = origin;
  document.getElementById('satisfaction').submit();
};
```

File: app/views/feedback.jade [hunk 3]
```diff
@@ -7,7 +7,29 @@ html
...
+        setInputValues();
+      });
+
+      function getQueryVariable(variable) {
+        var query = window.location.search.substring(1);
+        var vars = query.split('&');
+        for (var i=0; i<vars.length; i++) {
```
Comment (meddy) [inline app/views/feedback.jade]: you have some whitespace issues

File: app/views/feedback.jade [hunk 4]
```diff
@@ -7,7 +7,29 @@ html
...
+          var pair = vars[i].split('=');
+            if(pair[0] === variable){return pair[1];}
+        }
+        return(false);
```
Comment (meddy) [inline app/views/feedback.jade]: You don't need parens. Returning null is probably more semantic.

File: app/views/feedback.jade [hunk 5]
```diff
@@ -7,7 +7,24 @@ html
...
+        var satisfaction = getQueryVariable('satisfaction');
+        var origin = getQueryVariable('origin');
+        document.getElementById('result').value = satisfaction;
+        document.getElementById('origin').value = origin;
+        document.getElementById('satisfaction').submit();
+      }
+      function getQueryVariable(variable) {
+        var query = window.location.search.substring(1);
+        var vars = query.split('&');
+        for (var i=0; i<vars.length; i++) {
```
Comment (meddy) [inline app/views/feedback.jade]:
```
for (var i = 0;  i < vars.length; i++) {
 var pair = vars[i].split('=');
 if (pair[0] === variable) {
   return pair[1];
 }
}
```

**Comment by meddy** [comment]
🆒

File: app/views/feedback.jade [hunk 6]
```diff
@@ -7,7 +7,28 @@ html
...
+        var satisfaction = getQueryVariable('satisfaction');
+        var origin = getQueryVariable('origin');
+        document.getElementById('result').value = satisfaction;
+        document.getElementById('origin').value = origin;
+        document.getElementById('ip').value = myip;
+        document.getElementById('satisfaction').submit();
+      }
+      function getQueryVariable(variable) {
+        var query = window.location.search.substring(1);
+        var vars = query.split('&');
```
Comment (SamuelDavis) [inline app/views/feedback.jade]: why not just chain .split onto the substring and rename query to vars?
the query variable isn't used ever again

File: app/views/feedback.jade [hunk 7]
```diff
@@ -7,7 +7,28 @@ html
     meta(name='viewport' content='width=device-width, initial-scale=1')
 
     link(rel='stylesheet' href='styles/main.css')
-    link(rel='script' href='scripts/satisfaction.js')
+    script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
+    script.
+      window.onload = function () {
```
Comment (SamuelDavis) [inline app/views/feedback.jade]: why script tag everything instead of having a javascript file?

File: app/views/feedback.jade [hunk 8]
```diff
@@ -7,7 +7,28 @@ html
     meta(name='viewport' content='width=device-width, initial-scale=1')
 
     link(rel='stylesheet' href='styles/main.css')
-    link(rel='script' href='scripts/satisfaction.js')
+    script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
+    script.
```
Comment (SamuelDavis) [inline app/views/feedback.jade]: why script tag everything instead of having a javascript file?

**Comment by SamuelDavis** [comment]
:cake:

**Pull request merged**